### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MichalMaler**
Hello! Just prosing a little update for this well-written README based on my onboarding experience. :)

### CODE REVIEW

**Comment by inoxx03** [comment]
@Levi-Leah @adahms Would you please review Michal's updates to the README? They LGTM overall, but I'd like to get an ACK from you as well. Thank you:)

File: README.md [hunk 1]
```diff
@@ -2,12 +2,21 @@
...
+
+Alternatively, a user can clone this repository and add the following `pcmd` scripts on $PATH, but the ability to use predefined aliases will not be possible.
+
 ## Directory Structure
 
 The top level of this repository contains the following files and directories:
 
 **build**
-A directory that contains the latest RPM for the script, and the SPEC file used to generate the RPM.
+A directory that contains the:
```
Comment (adahms) [inline README.md:17]: s/A directory that contains the:/A directory that contains the following files:/

File: README.md [hunk 2]
```diff
@@ -2,12 +2,21 @@
 
 Pantheon CMD is a Python-based command-line tool that allows you to generate a rendered preview of modular documentation using the new HAML templates.
 
+Installing the Pantheon CMD tool using RPM allows users to perform calls using the predefined aliases such as:
```
Comment (adahms) [inline README.md:5]: s/the Pantheon CMD tool/Pantheon CMD/
s/allows users to perform calls/allows you to perform actions/

File: README.md [hunk 3]
```diff
@@ -2,12 +2,21 @@
 
 Pantheon CMD is a Python-based command-line tool that allows you to generate a rendered preview of modular documentation using the new HAML templates.
 
+Installing the Pantheon CMD tool using RPM allows users to perform calls using the predefined aliases such as: 
+* `pcmd validate`
+* `pcmd build`
+* `pcmd preview`
+
+Alternatively, a user can clone this repository and add the following `pcmd` scripts on $PATH, but the ability to use predefined aliases will not be possible.
```
Comment (adahms) [inline README.md:10]: s/a user can clone this/you can clone this/

File: README.md [hunk 4]
```diff
@@ -19,7 +28,7 @@ A directory containing the source files for the script, and the man page file.
 All additions and updates to the script are welcome.
 
 ## Packaging the Script
-After you update the manifest updater and test the changes, you can build an RPM-based package for the script so that it can be installed on systems that use *yum* or *dnf*.
+After you update the manifest updater and test the changes, you can build an RPM-based package for the script to be installed on systems that use *yum* or *dnf*.
```
Comment (adahms) [inline README.md:31]: s/the manifest updater/Pantheon CMD/

This one's on me - mind correcting this one for us at the same time? :)

File: README.md [hunk 5]
```diff
@@ -31,12 +40,15 @@ After you update the manifest updater and test the changes, you can build an RPM
    $ sudo dnf install subversion
    $ sudo dnf install rpm-build
    ```  
+2. Clone this repository.
```
Comment (adahms) [inline README.md:43]: @MichalMaler - how would you feel about adding a source section showing the clone command?

Does this mean we should also make it a prerequisite that a user has registered their SSH keys in GitHub?

File: README.md [hunk 6]
```diff
@@ -31,12 +40,15 @@ After you update the manifest updater and test the changes, you can build an RPM
...
 2. Open *./build/pantheon-cmd.spec*.
 3. Increment the value of the *Release* number.
+    As an example, `Release:   1%{?dist}` will increment the version of the build to `1.0.1`, where `{?dist}` acts as the identification of your Linux distribution.
```
Comment (adahms) [inline README.md:46]: s/will increment/increments/
s/acts as the identification of/identifies/

File: README.md [hunk 7]
```diff
@@ -31,12 +40,15 @@ After you update the manifest updater and test the changes, you can build an RPM
...
 4. Run the build script:
    ```shell
    $ sh make.sh 1.0
    ```
+   As a result, the `build/pantheon-cmd-1.0-X.<your-distribution-and-version>.noarch.rpm` file will be generated in the root of the repository. This file will be used in the following step.
```
Comment (adahms) [inline README.md:51]: s/will be/is/

**Review by adahms — CHANGES_REQUESTED** [review]
Thanks for the PR, @MichalMaler!

A few minor notes from a minimalism perspective, and one or two hopeful requests to fix a mistaken of my own in there, otherwise good to go. :)

File: README.md (hunk 4 above)
Comment (MichalMaler) [reply README.md:31]: WIth a pleasure :)

File: README.md (hunk 5 above)
Comment (MichalMaler) [reply README.md:43]: Good idea! Adding

**Comment by MichalMaler** [comment]
@adahms All applied! Can I ask for the second round? :)

**Review by adahms — APPROVED** [review]
LGTM; thanks, @MichalMaler!

**Comment by adahms** [comment]
Merging.

**Pull request merged**